tests/test_contrib.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow contrib."""
+
+import pytest
+from marshmallow import Schema, ValidationError, fields
+
+from marshmallow_utils.fields import contrib
```

Comment:
Add `Function` and `Method` to https://github.com/inveniosoftware/marshmallow-utils/blob/master/marshmallow_utils/fields/__init__.py to do that import switch from other PR comments :1st_place_medal:

Reply:
I have added those!